src/Route.js

```diff
@@ -41,6 +52,10 @@ Route.propTypes = {
   apiUrl: PropTypes.string.isRequired,
   loaderComponent: PropTypes.node,
   fetchClient: PropTypes.any,
+  options: PropTypes.objectOf(PropTypes.objectOf(PropTypes.shape({
```

Comment:
is there any other way to nest options proptypes ?

Reply:
there is - custom validator :)